dandi/tests/fixtures.py

```diff
@@ -307,6 +312,65 @@ def upload_dandiset(paths=None, **kwargs):
     }
 
 
+@pytest.fixture(scope="session")
+def download_video_files(tmp_path_factory):
+    from datalad.api import install
```

Comment:
Add in here:

```python
if shutil.which("git-annex") is None:
    pytest.skip("git-annex required")
```

That way, tests requiring git-annex will be skipped if it's not installed.